__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -147,11 +147,70 @@ exports[`linting javascript.js fixture matches snapshot 1`] = `
     "ruleId": "no-undef",
     "severity": 2,
   },
+  {
+    "column": 121,
+    "endColumn": 121,
+    "endLine": 45,
+    "fix": {
+      "range": [
+        894,
+        894,
+      ],
+      "text": "
+",
+    },
+    "line": 45,
+    "message": "Insert \`⏎\`",
+    "messageId": "insert",
+    "nodeType": null,
+    "ruleId": "prettier/prettier",
+    "severity": 2,
+  },
```

Comment:
This is because of this change:

https://github.com/Automattic/eslint-config-wpvip/pull/171/files#diff-0df5474287e5a16dc96f268437aaa2035aa854b4d43249f0ac20de228ca943eeR45 (removed EOL in __fixtures__/javascript.js)